bin/improver-standardise

```diff
@@ -0,0 +1,179 @@
+
+    parser.add_argument("--fix_float64", action='store_true', default=False,
+                        help="Check and fix cube for float64 data. Without "
+                             "this option a warning will be raised if float64 "
+                             "data is found but no fix applied.")
+
+    parser.add_argument("--check_float64", action='store_true', default=False,
+                        help="Check the cube for float64 data. If float64 "
+                             "data is found a warning will be raised but "
+                             "no fix applied.")
+
+    parser.add_argument("--regrid", action='store_true', default=False,
+                        help="regrid cube.....")
+
+    parser.add_argument("--nearest", action='store_true', default=False,
+                        help='If True, regridding will be performed using '
+                             'iris.analysis.Nearest() instead of Linear().'
+                             'Use for less continuous fields, '
+                             'e.g. precipitation.')
+
+    parser.add_argument("--extrapolation_mode", default='nanmask',
+                        help='Mode to use for extrapolating data into regions '
+                             'beyond the limits of the source_data domain. '
+                             'Modes are:'
+                             'extrapolate - The extrapolation points will '
+                             'take their value from the nearest source point. '
+                             'nan - The extrapolation points will be be '
+                             'set to NaN. '
+                             'error - A ValueError exception will be raised, '
+                             'notifying an attempt to extrapolate. '
+                             'mask  - The extrapolation points will always be '
+                             'masked, even if the source data is not a '
+                             'MaskedArray. '
+                             'nanmask - If the source data is a MaskedArray '
+                             'the extrapolation points will be masked. '
+                             'Otherwise they will be set to NaN. '
+                             'Defaults to nanmask.')
+
+    parser.add_argument("--json_file", metavar="JSON_FILE", default=None,
+                        help='Filename for the json file containing required '
+                             'changes to the metadata. Defaults to None.')
+
+    args = parser.parse_args()
+
+    # source file data path is a mandatory argument
+    source_data = load_cube(args.source_data_filepath)
+
+    # Re-grid with options:
+    # if a target grid file, output file path and regrid options have been
+    # specified, then regrid optionally applying float64 data check, metadata
+    # change, iris nearest and extrapolation mode as required
+    if args.target_grid_filepath and args.output_filepath and args.regrid:
```

Comment:
I see what you mean regarding the `regrid` option, because, as far as I can see, it is interchangeable with the `target_grid_filepath` option, as there's no other reason to specify a `target_grid_filepath`, other than for regridding. From that perspective, it might be best to remove the `regrid` option (sorry about that!) but edit the docstring of the `target_grid_filepath` to specify that if this option is supplied, then regridding is enabled.

Reply:
Changes made to do away with need for --regrid argument (implied by specifying target_grid_filepath)